samples/qa-a11y/src/views/Scroller.js

```diff
@@ -1,18 +1,90 @@
 /* eslint-disable react/jsx-no-bind */
 
 import Scroller from '@enact/sandstone/Scroller';
+import SpotlightContainerDecorator from '@enact/spotlight/SpotlightContainerDecorator';
+import Spottable from '@enact/spotlight/Spottable';
 import ToggleButton from '@enact/sandstone/SwitchItem';
 import Layout, {Cell} from '@enact/ui/Layout';
 import ri from '@enact/ui/resolution';
-import {useState} from 'react';
+import LS2Request from '@enact/webos/LS2Request';
+import {Fragment, useEffect, useState} from 'react';
+
+import css from './Scroller.module.less';
 
 const ScrollerView = () => {
-	const [native, setNative] = useState(true);
+	const [audioGuidance, setAudioGuidance] = useState(false);
+	const [audioGuidanceDisabled, setAudioGuidanceDisabled] = useState(true);
 	const [customAriaLabel, setCustomAriaLabel] = useState(false);
+	const [native, setNative] = useState(true);
 	const scrollMode = native ? 'native' : 'translate';
 
+	useEffect( () => {
+		if (window.WebOSServiceBridge ?? window.PalmServiceBridge) {
+			new LS2Request().send({
+				service: 'luna://com.webos.settingsservice/',
+				method: 'getSystemSettings',
+				parameters: {
+					category: 'option',
+					keys: ['audioGuidance']
+				},
+				onSuccess: () => {
+					setAudioGuidanceDisabled(false);
+				}
+			});
+		}
+	}, []);
+
 	const handleChangeJSNativeButton = () => setNative(!native);
 	const handleChangeAriaLabelButton = () => setCustomAriaLabel(!customAriaLabel);
+	const handleChangeAudioGuidance = () => {
```

Comment:
This function breaks my previous qa-a11y test cases. 
1. Turn on audio guidance from the settings app
2. run qa-a11y app
3. select Scroller menu
4. Test the focused scrollbar
5. switch on the Audio Guidance 
6. Test the new Scroller you added (non-focusable scrollbar)
7. switch off the Audio Guidance
8. How can I check the original test number 5?

Reply:
I removed audio guidance call from the sample code.

Reply:
The updated code looks OK.